src/Compiler.ts

```diff
@@ -98,9 +98,16 @@ export class Compiler {
     const { tsConfig, tsConfigFile } = await this.getTSConfig(filename);
     if (this.tsConfigMap[tsConfigFile]) return;
 
+    let fileNames: string[];
+    if (process.platform === "win32") {
+      fileNames = tsConfig.fileNames.map(e => e.replace(/\//g, "\\"));
```

Comment:
Strange that TypeScript doesn't hand them out with the right separator.

Reply:
It looks like TypeScript always hands out filenames with `/` separator.